### PR TITLE
[documentation]: Review language-specific guides

### PR DESCRIPTION
**Pull request by outoftardis**

### CODE REVIEW

File: src/guide/rust.md [hunk 1]
```diff
@@ -326,8 +338,7 @@ iroha_client.submit(register_time)?;
 
 This creates an asset `time` that can only be minted once and has the type
 `fixed`. `AssetDefinition::fixed` just like its other cousins (`quantity`
-and `big_quantity`) returns a builder of an `AssetDefinition`. If you get
-odd error messages, that's because you didn't call `build()` at the end.
```
Comment (outoftardis) [inline src/guide/rust.md:330]: I think this part is unnecessary due to #53

File: src/guide/python.md [hunk 2]
```diff
@@ -8,37 +8,37 @@ WIP: `iroha-python` works only with `iroha v2-pre-rc.1` for now.
```
Comment (outoftardis) [inline src/guide/python.md:8]: @appetrosyan line 5: is that still true?

File: src/guide/bash.md [hunk 3]
```diff
@@ -1,70 +1,87 @@
...
-might use the excellent Python libraries for creating a useful TUI. For
-now, one must make do with what one has.
+need to put in.
+
+::: info
+
+It is possible that in the future, once `iroha_client_cli` is phased out
+and `iroha_python` is the official scripting interface, we might use the
+excellent Python libraries for creating a useful TUI. For now, one must
+make do with what one has.
```
Comment (WRRicht3r) [inline src/guide/bash.md:69]: 🫶

File: src/guide/rust.md [hunk 4]
```diff
@@ -411,6 +428,6 @@ for event in iroha_client.listen_for_events(filter)? {
 };
 ```
 
-**Needless to say that a synchronous infinite blocking loop is bad UX for
-anything but a command-line program**, but for illustration purposes, this
+Needless to say that a synchronous infinite blocking loop is bad UX for
```
Comment (WRRicht3r) [inline src/guide/rust.md:431]:
```suggestion
Needless to say, an synchronous infinite blocking loop is bad UX for
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/guide/kotlin-java.md [hunk 5]
```diff
@@ -9,12 +9,12 @@ the unusual, and provide some instructions for creating your own Iroha
...
 
 We assume that you know how to create a new package and have basic
-understanding of the fundamental kotlin code. Specifically, we shall assume
+understanding of the fundamental Kotlin code. Specifically, we shall assume
 that you know how to build and deploy your program on the target platforms.
 The Iroha 2 JVM-compatible SDKs are as much a work-in-progress as the rest
 of this guide, and significantly more so than the Rust library.
 
-Without further ado, here’s an example `build.gradle` file.
+Without further ado, here's an example `build.gradle` file:
```
[the remark this answers is not in the document]
Comment (outoftardis) [reply src/guide/kotlin-java.md:17]:
```suggestion
Without further ado, here's the example `build.gradle` file:
```

**Review by WRRicht3r — APPROVED** [review]

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**